pkg/scheduler/algorithmprovider/defaults/defaults.go

```diff
@@ -255,6 +251,11 @@ func defaultPriorities() sets.String {
 
 		// Prioritizes nodes that marked with taint which pod can tolerate.
 		factory.RegisterPriorityFunction2("TaintTolerationPriority", priorities.ComputeTaintTolerationPriorityMap, priorities.ComputeTaintTolerationPriorityReduce, 1),
+
+		// ImageLocalityPriority prioritizes nodes based on locality of images requested by a pod. Nodes with larger size
+		// of already-installed packages required by the pod will be preferred over nodes with no already-installed
+		// packages required by the pod or a small total size of already-installed packages required by the pod.
+		factory.RegisterPriorityFunction2("ImageLocalityPriority", priorities.ImageLocalityPriorityMap, nil, 1),
```

Comment:
[Update] According to discussion in sig-scheduling, we need to work out a proper weight (vs default weight) for this priority in case some heat nodes are always chosen.

Reply:
@k82cn Please check the benchmark posted in sig meeting note.